### ISSUE TITLE
Can you do a transmit based fm

### ISSUE THREAD

**Issue by esgx001** [demantz/hackrf_android#13]

**Comment by demantz** [comment]
Are you talking about a FM transmitter app like this:
select music/mic
send as FM signal to be received by an off-the-shelf radio?

I thought about that but I will rather do a general transmit app at some time..

**Comment by esgx001** [comment]
of course, a general app is more cool, I mean it can includes the function i said. it's basic, then you can extend the general app. finish the app,with the app,we can communicate with the intercom

**Comment by esgx001** [comment]
i am learning the tx based on fm on android, wish i can learn the code about transmitting of you ,thanks

**Comment by Radiomix2000** [comment]
You need some DSP skills, to make software FM.

**Comment by pavlus** [comment]
Not so much skills needed to implement modulation, much more to integrate full pipeline from recording, modulating, interpolating and transmitting them to device + UI for that, into existing app, so it could be extended later.
